i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md

```diff
@@ -5,147 +5,380 @@ title: Cisco SSMS
 tout en donnant aux organisations sensibles à la sécurité un moyen d'accéder à un sous-ensemble de fonctionnalités de Cisco SSM
 sans utiliser de connexion Internet directe pour gérer leur base d'installation.
 
-## Contenu du connecteur de supervision
+## Contenu du pack
+
+### Modèles
+
+Le connecteur de supervision **Cisco SSMS** apporte un modèle d'hôte :
+
+* **App-Cisco-Ssms-Restapi-custom**
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="App-Cisco-Ssms-Restapi-custom" label="App-Cisco-Ssms-Restapi-custom">
+
+| Alias    | Modèle de service                      | Description                               |
+|:---------|:---------------------------------------|:------------------------------------------|
+| Alerts   | App-Cisco-Ssms-Alerts-Restapi-custom   | Contrôle les alertes mineures et majeures |
+| Licenses | App-Cisco-Ssms-Licenses-Restapi-custom | Contrôle les licences                     |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Cisco-Ssms-Restapi-custom** est utilisé.
 
-### Objets supervisés
+</TabItem>
+</Tabs>
 
-* Applications
-* Serveurs
-* Licences
-* Alertes
+### Métriques & statuts collectés
 
-### Métriques collectées
+Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
 
 <Tabs groupId="sync">
 <TabItem value="Alerts" label="Alerts">
 
-| Metric Name                | Description              |
-| :------------------------- | :------------------------|
-| account.alerts.minor.count | Number of alerts minor.  |
-| account.alerts.major.count | Number of alerts major.  |
+| Métrique                              | Unité |
+|:--------------------------------------|:------|
+| *accounts*~account.alerts.minor.count | count |
+| *accounts*~account.alerts.major.count | count |
+| *accounts*~*alerts*#alert-status      | N/A   |
 
 </TabItem>
 <TabItem value="Licenses" label="Licenses">
 
-| Metric Name                | Description                            |
-| :------------------------- | :------------------------------------- |
-| licenses.usage.count       | Number of licenses usage.              |
-| licenses.free.count        | Number of licenses free.               |
-| licenses.usage.percentage  | Percentage of licenses usage. Unit : % |
+| Métrique                                        | Unité |
+|:------------------------------------------------|:------|
+| *accounts*~*licenses*#license-status            | N/A   |
+| *accounts*~*licenses*#licenses.usage.count      | count |
+| *accounts*~*licenses*#licenses.free.count       | count |
+| *accounts*~*licenses*#licenses.usage.percentage | %     |
 
 </TabItem>
 </Tabs>
 
 ## Prérequis
 
 Un compte de service est requis pour interroger l'API Cisco SSMS. Celui-ci doit avoir suffisamment de privilèges en lecture du compte ciblées.
```

Comment:
```suggestion
Un compte de service est requis pour interroger l'API Cisco SSMS. Celui-ci doit disposer de privilèges de lecture suffisants pour le compte ciblé.
```
Does it can be more relevant according to the english version : A service account is required to query the Cisco SSMS API. It must have sufficient read privileges in the account targeted.